modules/ROOT/nav.adoc

```diff
@@ -17,40 +17,51 @@ include::{partialsdir}/attributes.adoc[]
 ** xref:configuring-dev-env.adoc[Configuring a Dev Environment]
 
 * Using Mobile Services
+
 // PUSH
 ** xref:push/index.adoc[{unifiedpush-service}]
 *** xref:push/provisioning.adoc[Provisioning]
 *** xref:push/binding.adoc[Binding]
-*** xref:configuring-dev-env.adoc[Configuring a Dev Environment]
+*** xref:push/configuring-dev-env-push.adoc[Configuring a Dev Environment]
 *** xref:push/coding.adoc[Adding Code]
 *** xref:push/admin-ui.adoc[{push-ui}]
 *** More...
 **** xref:push/proc_sending-a-push-notification.adoc[Sending a Push Notification]
 **** xref:push/handling-push-notifications.adoc[Handling Push Notifications]
 **** xref:push/proc_preparing-ups-for-use-with-ios.adoc[Apple Push Network]
 **** xref:push/google-setup.adoc[Firebase Cloud Messaging]
+
```

Comment:
@jstaffor these blank lines will break the structure in the left hand menu, it has happened to me before.

Reply:
Thanks Jose - addressed in 857140205782ec7da1be2900b3635904ca5df4e0